enterprise/coderd/license/license_test.go

```diff
@@ -378,6 +378,53 @@ func TestEntitlements(t *testing.T) {
 		}
 	})
 
+	t.Run("AllFeaturesAlwaysEnable", func(t *testing.T) {
+		t.Parallel()
+		db := dbfake.New()
```

Comment:
Should we change this to `dbtestutil.NewDB` so that we also test with postgres?

Reply:
These are unit tests, not integration.